RUFAS/weather.py

```diff
@@ -16,33 +15,23 @@ class Weather:
     """
     The `Weather` class manages all weather data used to run a single simulation.
 
+    Parameters
+    ----------
+    weather_file : dict[str:List[Any]]
```

Comment:
```suggestion
    weather_file : dict[str, List[Any]]
```
just to be consistent with the rest of the codebase